python/fusion_engine_client/analysis/analyzer.py

```diff
@@ -1335,59 +1335,57 @@ def plot_wheel_data(self):
         if self.output_dir is None:
             return
 
-        self._plot_wheel_ticks_or_speeds(type='speed')
-        self._plot_wheel_ticks_or_speeds(type='tick')
+        self._plot_wheel_ticks_or_speeds(source='wheel', type='speed')
+        self._plot_wheel_ticks_or_speeds(source='wheel', type='tick')
+        self._plot_wheel_ticks_or_speeds(source='vehicle', type='speed')
+        self._plot_wheel_ticks_or_speeds(source='vehicle', type='tick')
 
-    def _plot_wheel_ticks_or_speeds(self, type):
+    def _plot_wheel_ticks_or_speeds(self, source, type):
         """!
         @brief Plot wheel speed or tick data.
         """
         # Read the data. Try to determine which type of wheel output is present in the log (if any).
```

Comment:
This really needs a table to understand what the intent of this is. If I understand, this is to support a few different things:
1. This function is meant to be reused between ticks/speed
2. ticks and speed can be for either the wheels or the vehicle
3. there are some different messages that may be used for backwards compatibility
4. some messages support a separate raw and corrected version 
5. there are some different messages that may be used based on how the input was sent to the device in realtime
6. messages may or may not be enabled for the log

Maybe referencing this list of considerations, or a representative example would make this a bit easier to parse.